modules/database/src/interfaces/CmsOptions.ts

```diff
@@ -0,0 +1,33 @@
+export interface CmsOptions {
+  enabled: boolean;
+  authentication: boolean;
+  crudOperations: {
+    [key: string]: {
```

Comment:
Don't drop the key signature. I only requested that the inner type be removed (if unused).
Switch back to a typed object for `crudOperations` keeping the `typeof` inference.